packages/abi-gen-wrappers/src/generated-wrappers/static_call_proxy.ts

```diff
@@ -26,30 +26,33 @@ import * as ethers from 'ethers';
 /* istanbul ignore next */
 // tslint:disable:no-parameter-reassignment
 // tslint:disable-next-line:class-name
-export class IValidatorContract extends BaseContract {
```

Comment:
`abi-gen-wrappers` contains both IValidatorContract and `StaticCallProxyContract`. git is confusing the diffs.